src/01-general/users.apib

```diff
@@ -18,7 +18,22 @@ Get the current authenticated user.
             + last_name: `Smith` (string)
             + email: `john@teamleader.eu` (string)
             + telephones (array[Telephone])
-            + language: `nl` (string)
+            + language: `nl-BE` (enum[string])
+                + Members
+                    + da-DK
+                    + de-DE
+                    + en-GB
+                    + es-ES
+                    + fi-FI
+                    + fr-FR
+                    + it-IT
+                    + nb-NO
+                    + nl-BE
+                    + nl-NL
+                    + pl-PL
+                    + pt-PT
+                    + sv-SE
+                    + tr-TR
```

Comment:
It might be useful to define this list in one place so it becomes re-usable? This might become a maintainability problem if it is used in more places.